### ISSUE TITLE
Please consider renaming / rebranding Edge redirector script

### ISSUE THREAD

**Issue by SamHills** [AveYo/fox#17]
Your program is just what I needed.  But I have an issue with the name.
Even if you try to obfuscate a vulgar or offensive word by leaving out or changing some of the letters, it's the same word and is just as vulgar and offensive as the original word.
I suggest "ChrEdgeBuzzOff".  "Buzz off" means the same thing as the phrase you used, and it isn't vulgar.
I hate Micro$$$oft browsers as much as you apparently do.  But I can't include anything that uses vulgar offensive language, even if it's thinly obfuscated.

**Comment by AveYo** [comment]
**Microsoft Edge Chromium Based Browser and the way it's pushed IS vulgar.**
How about not living in Victorian era fake-decency that's all about semantics and none about acting.

That said, I did consider a rebrand even before you come up with an impolite, personal-attack named issue instead of being about the script. Here, let me rephrase _"Please don't be vulgar #17"_ for you:
"Please consider renaming / rebranding Edge redirector script".

**Comment by mc510** [comment]
I just want to say that I love the name of this script! The script not only fixes the problematic behavior, it also captures and represents my feelings about it. Perfection! For anyone who has a problem with the name of the script, I'd point out that nobody is forcing them to use it (or "include" it, whatever is meant by that), and they even have to opportunity to fork the project and rename it whatever they want for their own use. Perhaps something like "EdgeWouldYouKindlyBackOffTYSM.cmd"

**Comment by AveYo** [comment]
Rebranded to `OpenWebSearch`. I still read it as `ChrEdgeFkOff` ;)
